From 6fc1fdf73a6fd5fcd4d1f7b5e3fe74dd230f6243 Mon Sep 17 00:00:00 2001
From: "Wesley.CW Wang" <wesleycwwang@google.com>
Date: Tue, 6 Nov 2018 17:30:04 +0800
Subject: [PATCH 7/9] Disable EmergencyDialer multi window mode

- Set android:resizeableActivity=false to disable multi window.

Test: Manually
Bug: 119078730
Change-Id: I55350678b394626aadc64d1f71614b3b864b6a42
---
 AndroidManifest.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 780ae2cc0..c940b487e 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -224,7 +224,8 @@
         <activity android:name="EmergencyDialer"
             android:label="@string/emergencyDialerIconLabel"
             android:theme="@style/EmergencyDialerTheme"
-            android:screenOrientation="portrait">
+            android:screenOrientation="portrait"
+            android:resizeableActivity="false">
             <intent-filter>
                 <action android:name="com.android.phone.EmergencyDialer.DIAL" />
                 <category android:name="android.intent.category.DEFAULT" />
-- 
2.17.1

